tedana/workflows/tedana.py

```diff
@@ -109,32 +104,28 @@ def _get_parser():
                               'logcosh (default), cube, exp'),
                         choices=['logcosh', 'cube', 'exp'],
                         default='logcosh')
-    parser.add_argument('--denoiseTEs',
-                        dest='dne',
+    parser.add_argument('--verbose',
+                        dest='verbose',
                         action='store_true',
-                        help='Denoise each TE dataset separately.',
+                        help='Generate intermediate and additional files.',
                         default=False)
-    parser.add_argument('--strict',
-                        dest='strict',
+    parser.add_argument('--gscontrol',
```

Comment:
I'm not sure why we have both  `gscontrol` and `global_denoise` arguments. Could you explain what's the benefit, here ?

Reply:
GSR (`gscontrol_raw`) and T1c-GSR (`gscontrol_mmix`) are both run by default on the data, rather than treated as competing alternatives to one another. Also, since GSR is run within the pipeline, while T1c-GSR (and someday GODEC, dGSR with rapidtide, etc.) is run after, as a sort of post-processing step.

That said, we can clean this up. Maybe we could allow multiple global denoising steps? E.g.,
`--global_denoise gsr t1c` if you want to run both.

Reply:
Yes, I think I'd prefer that ! Since they're both aimed at the same outcome, it would make more sense to group them together, in my mind. WDYT ?

Reply:
I can do that, but I'm still a little wary of the argument name. Can you think of anything better that `global_denoise`, `denoise`, or `ws_denoise`?

Reply:
I liked `gs_control` ! Since it's "controlling" global signal, through various means :)

Reply:
That works, I think. The other methods we'd ultimately consider include rPCA, GODEC, dynamic GSR with rapidtide, aCompCor, and tCompCor. The only ones where `gs_control` (maybe `gscontrol`?) would seem kind of weird are aCompCor and tCompCor, but it's not too much of a stretch.

Reply:
Also, if we're integrating into larger pipelines, many of those already do aCompCor and tCompCor !

Reply:
That's true, and aCompCor actually can't be integrated into tedana without information we don't want to require, although I imagine that a number of denoising steps will need to go _inside_ of tedana the same way that GSR and T1c-GSR are done.